charts/verdaccio/templates/deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     metadata:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/htpasswd-secret: {{ include (print $.Template.BasePath "/htpasswd-secret.yaml") . | sha256sum }}
```

Comment:
I may be better to hash `.Values.secrets.htpasswd` than the secret because the `htpasswd` function returns different output every time you run `helm upgrade` even though the actual users/passwords didn't change.

`checksum/htpasswd: {{ sha256sum .Values.secrets.htpasswd }}`

Reply:
This way the checksum will only change when the actual user/passwords change and not because `htpasswd` returned different output for the same input for every `helm upgrade`